compiler/src/main/scala/org/scalajs/nscplugin/GenJSCode.scala

```diff
@@ -1073,7 +1071,8 @@ abstract class GenJSCode[G <: Global with Singleton](val global: G)
       !settings.noForwarders && sym.isStatic && !isImplClass(sym) && {
         // Reject non-top-level objects unless opted in via the appropriate option
         scalaJSOpts.genStaticForwardersForNonTopLevelObjects ||
-        !sym.name.containsChar('$') // this is the same test that scalac performs
+        !sym.name.containsChar(
+            '$') // this is the same test that scalac performs
```

Comment:
These comment moves are unfortunate, not sure we can do anything about them :(

Reply:
For really long lines, there's nothing to be done, no. This example, however, is fixed by increasing max line length.

Reply:
have you tried https://scalameta.org/scalafmt/docs/configuration.html#newlinesavoidforsimpleoverflow?